package.json

```diff
@@ -16,7 +16,9 @@
     "eslint": "^8.9.0"
   },
   "devDependencies": {
-    "eslint": "^8.9.0"
+    "eslint": "^8.9.0",
+    "@semantic-release/git": "^10.0.1",
+    "semantic-release": "^18.0.1"
```

Comment:
This is a major version older than the one I've been running.  Is there a reason to use this one?